dfdaemon/config/config.go

```diff
@@ -364,7 +364,7 @@ func certPoolFromFiles(files ...string) (*x509.CertPool, error) {
 	return roots, nil
 }
 
-// Proxy describe a regular expression matching rule for how to proxy a request
+// Proxy describes a regular expression matching rule for how to proxy a request
```

Comment:
I think there should be a dot at the end of comment `// Proxy describes a regular expression matching rule for how to proxy a request ` -> `// Proxy describes a regular expression matching rule for how to proxy a request . `,it will be better? the other comment are the same

Reply:
I think the same as you think, but there are too many comments that don't have the dot.
I am going to add the dot together for these comments in the next pr.